### PR TITLE
Decrease verbosity ccdb api

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: CCDB/test/testCcdbApi.cxx [hunk 1]
```diff
@@ -164,6 +164,10 @@ BOOST_AUTO_TEST_CASE(store_test, *utf::precondition(if_reachable()))
   auto h1 = new TH1F("object1", "object1", 100, 0, 99);
   h1->FillRandom("gaus", 10000);
   f.api.store(h1, "Test/Detector", f.metadata, -1, -1, true);
+
+  auto h2 = new TH1F("object2", "object2", 100, 0, 99);
+  h1->FillRandom("gaus", 10000);
+  f.api.store(h1, "Test/Detector", f.metadata, -1, -1, true);
```
Comment (sawenzel) [inline CCDB/test/testCcdbApi.cxx]: should this be `h2` in the last 2 lines (or what is the purpose of h2?)?

Comment (Barthelemy) [reply CCDB/test/testCcdbApi.cxx]: you are absolutely correct. My copy pasting capabilities are decreasing.

File: CCDB/src/CcdbApi.cxx [hunk 2]
```diff
@@ -27,6 +27,8 @@
 #include <TWebFile.h>
 #include <TH1F.h>
 #include <TTree.h>
+#include <FairLogger.h>
+#include <TError.h>
```
Comment (sawenzel) [inline CCDB/src/CcdbApi.cxx]: Why do you need TError? (FairLogger defines its own ERROR macro I think).

Comment (Barthelemy) [reply CCDB/src/CcdbApi.cxx]: I need it to decrease the verbosity of ROOT itself during the reading of the TFile.

Comment (sawenzel) [reply CCDB/src/CcdbApi.cxx]: That's what I figured out in between. So it's already merged!

**Pull request merged**